cmd/kpod/login.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"golang.org/x/crypto/ssh/terminal"
+
+	"github.com/containers/image/pkg/authentication"
```

Comment:
Please link to a corresponding PR to containers/image with this dependent pkg?

Reply:
https://github.com/containers/image/pull/333